_test/test_sqw/test_save.m

```diff
@@ -42,14 +44,58 @@
                 u, v, psi, omega, dpsi, gl, gs,...
                 [10,5,5,5]);
             obj.sqw_obj = obj.sqw_obj{1};
+            hp = horace_paths;
+            sqw_array_file = fullfile(hp.test,'test_TF_components',obj.sqw_array_file);
+            ld = load(sqw_array_file);
+            obj.sqw_array =ld.fe_arr;
+        end
+        %------------------------------------------------------------------
+        function test_save_array_names_template_provided(obj)
+            files_to_save = {
+                fullfile(tmp_dir,'fe_1.sqw'),...
+                fullfile(tmp_dir,'fe_2.sqw'),...
+                fullfile(tmp_dir,'fe_3.sqw')};
+            clOb = onCleanup(@()del_memmapfile_files(files_to_save ));
+            obj.sqw_array.save( fullfile(tmp_dir,'fe.sqw'));
+            assertTrue(isfile(files_to_save{1}))
+            assertTrue(isfile(files_to_save{2}))
+            assertTrue(isfile(files_to_save{3}))
+        end
 
+        function test_save_array_names_provided(obj)
+            files_to_save = {
+                fullfile(tmp_dir,'fe1.sqw'),...
+                fullfile(tmp_dir,'fe2.sqw'),...
+                fullfile(tmp_dir,'fe3.sqw')};
```

Comment:
may be worth making the names A B C rather than 1 2 3 so no overlap with the template. For both tests, may be worth unpacking the files and equating with the original objects